gst/nnstreamer/tensor_query/nnstreamer-edge-internal.h

```diff
@@ -42,13 +42,17 @@ typedef struct {
   bool is_server;
   int64_t client_id;
   char *caps_str;
-  char *recv_ip;
-  int recv_port;
+  char *listner_ip;
+  int listner_port;
   GHashTable *conn_table;
   GSocketListener *listener;
 
   /* MQTT */
   void *mqtt_handle;
+  GAsyncQueue *server_list;
```

Comment:
how about adding new struct for mqtt?
Suggestion: nns_edge_mqtt_state_s in -mqtt.c / nns_edge_broker_handle in common.h

Reply:
Or.. will you update these on nnstreamer-edge repo?

Reply:
Ok, I will move mqtt data structure to -mqtt.c.